jobs/rocoto/arch.sh

```diff
@@ -12,12 +12,6 @@
 ## cyc    : current cycle (HH)
 ###############################################################
 
-###############################################################
-# Source FV3GFS workflow modules
-. $HOMEgfs/ush/load_fv3gfs_modules.sh
```

Comment:
This will be required on the R&D machines to load HPSS module, no?
Can you provide a reason why this is deleted?

Reply:
NCO does not use this script in any form whatsoever.  Please correct me if I am incorrect.

Reply:
why are we changing rocoto scripts in this PR? are they not working for rocoto ?

Reply:
NCO removed scripts source $HOMEgfs/ush/load_fv3gfs_modules.sh due to environment variable been overwrite to default that break ecflow workflow.